pkgs/applications/networking/browsers/firefox/packages.nix

```diff
@@ -94,6 +96,7 @@
 
   firefox-esr-115 = buildMozillaMach rec {
     pname = "firefox-esr-115";
+    binaryName = "firefox-esr";
```

Comment:
There is a short time where two ESR versions will overlap. Would be good to disambiguate them through their major version.